elements/pfe-codeblock/README.md

```diff
@@ -0,0 +1,72 @@
+# PatternFly Element | Codeblock element
+Render code in a styled and fromatted way
+
+## Usage
+Describe how best to use this web component along with best practices.
+
+```html
```

Comment:
Can you use one of the codeblocks from the demo page for this? Maybe two examples? The HTML one and the JS one might be good here.